Code/skyrim_ui/src/app/components/settings/settings.component.html

```diff
@@ -129,11 +129,11 @@ <h2>{{ 'COMPONENT.SETTINGS.PARTY' | transloco }}</h2>
       <h2>{{ 'COMPONENT.SETTINGS.DEBUG' | transloco }}</h2>
       <div class="flex-item">
         <label for="showDebug">{{ 'COMPONENT.SETTINGS.SHOW_DEBUG_INFO' | transloco }}</label>
-        <app-checkbox
+        <input type="checkbox"
```

Comment:
put it in a new line for consistency
```suggestion
        <input 
          type="checkbox"
```